kolibri/logger/migrations/0001_initial.py

```diff
@@ -56,7 +58,7 @@ class Migration(migrations.Migration):
                 ('start_timestamp', models.DateTimeField()),
                 ('end_timestamp', models.DateTimeField(blank=True, null=True)),
                 ('time_spent', models.FloatField(default=0.0, help_text='(in seconds)', validators=[django.core.validators.MinValueValidator(0)])),
-                ('progress', models.FloatField(default=0, validators=[django.core.validators.MinValueValidator(0)])),
+                ('progress', models.FloatField(default=0, validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(1.0)])),
```

Comment:
I don't think we should be modifying existing migrations...?

Reply:
That's an "old" discussion :) Depends on whether you want the extra meaningless migration :)

Reply:
We should come up with a policy on this. Until now, I believe we've generally tried to avoid modifying existing migrations though we may have deviated from that practice occasionally.